### PR TITLE
Added setup.py for installing Python bindings

### PR DESCRIPTION
**Pull request by simondlevy**
This feature allows you to install the Python cffirmware lib globally.

### CODE REVIEW

**Comment by knmcguire** [comment]
Looks all good to me! @whoenig any objections to this?

File: docs/building-and-flashing/build.md [hunk 1]
```diff
@@ -137,9 +137,10 @@ First make sure that you have [SWIG](https://swig.org/) installed on your system
 ```bash
 make cf2_defconfig
 make bindings_python
+cd build
+sudo python3 setup.py install
```
Comment (whoenig) [inline docs/building-and-flashing/build.md:141]: I would prefer `python3 setup.py install --user`, if that works?

**Review by whoenig — APPROVED** [review]
I like it generally, but it would be better if we could avoid the sudo somehow. I suggested a possibility inline.

**Comment by simondlevy** [comment]
I agree with @whoenig about the sudo.  I am about to do another pull request that uses --user instead

**Comment by simondlevy** [comment]
I may have messed this whole process up, but it should be straightforward to get the two changes in anyway.

**Comment by knmcguire** [comment]
Alright! We will keep an eye out on the new PR then :)

**Pull request closed without merging**